easy_run.py

```diff
@@ -48,7 +48,7 @@ def initialize_api():
         f.writelines(keys)
         f.close()
     else:
-        if keys[0] == "Replace THIS line with the Todoist API token. Remove trailing spaces\n" :
+        if keys[0] == "Replace THIS liane with the Todoist API token. Remove trailing spaces\n" :
```

Comment:
Should stay as "line"

Reply:
Thanks, good catch. Appreciate the help!